io/io/test/CMakeLists.txt

```diff
@@ -5,7 +5,10 @@
 # For the list of contributors see $ROOTSYS/README/CREDITS.
 
 ROOT_ADD_GTEST(RRawFile RRawFile.cxx LIBRARIES RIO)
-ROOT_ADD_GTEST(TFile TFileTests.cxx LIBRARIES RIO)
+ROOT_ADD_GTEST(TFile TFileTests.cxx
+  LIBRARIES RIO
+  COPY_TO_BUILDDIR ${CMAKE_CURRENT_SOURCE_DIR}/MySubClassUnv.cxx
```

Comment:
We tend to have the "complex" tests be placed in roottest.   Consider moving (just the new part of) the test to `roottest/root/io/evolution/versions